docs/logs/fields.md

```diff
@@ -4,4 +4,68 @@ description: Sematext Logs App looks for special fields in logs, namely os.host,
 
 ## Common Schema
-Fields in log events are also referred to as Tags in Sematext.  They are used for searching and filtering, but also for pivoting from Logs to other observability data in Sematext, such as performance metrics in [Monitoring](../monitoring).  The [Common Schema for Logs](../tags/common-schema/#logs-tags) lists special fields and their meaning.
+Fields in log events are also referred to as Tags in Sematext. They are used for searching and filtering, but also for pivoting from Logs to other observability data in Sematext, such as performance metrics in [Monitoring](../monitoring).  The [Common Schema for Logs](../tags/common-schema/#logs-tags) lists special fields and their meaning.
+
+## Fields Structure
+
+The structure of your data is dictated by the data shipped to Sematext Logs as log events. There are two places where you can easily see your data structure - the fields and filters section of your Logs table and the [Field Editor](/logs/fields/#field-editor):
+
+<img src="../../images/logs/logs_structure_fields_and_filters.png" alt="Logs Fields and Filters">
+
+<img src="../../images/logs/logs_structure_field_editor.png " alt="Logs Field Editor">
+
+---
+**Note:**
+The structure shown by the fields and filters section and the field editor may differ. The data structure shown by the fields and filters always covers the **whole data** - old and new. The data structure shown by the Field Editor is always the **current** one, the one that will be applied to all your current and future events.
+--- 
+
+### Field Types
+
+Each field has a field type associated with it and will include:
+
+ * [Integer/Long](/logs/field-types/#integerlong) - numerical data without floating points
+ * [Float/Double](/logs/field-types/#floatdouble) - numerical, floating point data
+ * [Boolean](/logs/field-types/#boolean) - Boolean field
+ * [Keyword/Not analyzed string](/logs/field-types/#not-analyzed-string) - not analyzed text
+ * [Text/Analyzed string](/logs/field-types/#analyzed-string) - analyzed, full text searchable data
+ * [Date](/logs/field-types/#date) - date based data
+ * [Geo](/logs/field-types/#geo) - field type dedicated for spatial data
+ * Object - nested structure holding structured data
+
+### Modifying Fields
+
+You can [add](/logs/fields/#adding-fields), [remove](/logs/fields/#removing-fields) or [edit](/logs/fields/#editing-fields) existing fields by using the [field editor] accessible via a Logs App settings or by using the [Templates and Mappings](/logs/mappings-templates) APIs directly. Keep in mind that the modification is applied only to the data that is shipped after the adjustments, so you may need to re-index the data in case you want the old data to be adjusted. 
+
+## Field Editor
+
+The Field Editor functionality allows adding, editing, and removing fields present in your data structure definition.
```

Comment:
> data structure definition

What's that? ;) I mean, I know what you are referring to, but this "data" we keep mentioning is really "logs", so it may be better to remove that layer of abstraction.  And "structure definition" is what we refer to as mapping?

Reply:
Good point, adjusted :)